src/app/aircraft/edit/post.controller.js

```diff
@@ -34,13 +34,22 @@ module.exports = (req, res) => {
       // call the API to update craft
       craftApi.update(craftReg, craftType, craftBase, cookie.getUserDbId(), craftId)
         .then((apiResponse) => {
-          const parsedResponse = JSON.parse(apiResponse);
-          if (Object.prototype.hasOwnProperty.call(parsedResponse, 'message')) {
-            // API returned failure
-            res.render('app/aircraft/edit/index', { cookie, errors: [parsedResponse] });
-          } else {
-            // API returned successful
-            res.redirect('/aircraft');
+          try {
```

Comment:
This is a temporary solution to an issue where the app would return an unhelpful error if they attempted to rename an aircraft registration to one already in the DB.

This needs fixing properly by correcting the index in the DB to make user_id and registration or org_id and registration the unique indices.